client/components/nav.jsx

```diff
@@ -1,17 +1,23 @@
 import React from 'react';
 
-export default function Nav(props) {
+export default class Nav extends React.Component {
 
-  return (
+  onClick() {
+    window.location.hash = '#form';
+  }
+
+  render() {
+    return (
     <>
       <nav className="navbar navbar-dark bg-primary fixed-top">
-        <div className="container justify-content-start">
-          <button className="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarToggleExternalContent" aria-controls="navbarToggleExternalContent" aria-expanded="false" aria-label="Toggle navigation">
-            <span className="navbar-toggler-icon"></span>
-          </button>
-          <a className="navbar-brand mx-3" href="#">Memento</a>
+        <div className="container px-3">
+          <a className="navbar-brand" href="#">Memento</a>
+            <button className="btn btn-primary" type="button" onClick={this.onClick}>
+              <i className="fa fa-plus" />
+            </button>
```

Comment:
Instead of a button with an `onClick`, it's better to make this an anchor with `href` and style it link a button

https://getbootstrap.com/docs/5.2/components/buttons/#button-tags